app/models/random_access_key.rb

```diff
@@ -0,0 +1,7 @@
+class RandomAccessKey < ApplicationRecord
+  belongs_to :request
+
+  after_initialize do |key|
```

Comment:
@syncrou @eclarizio Is there a better callback to use here?  I'm concerned we are doing this ever time the object is instantiated (which is protected be the `||=`) but it would be nice to avoid that.  I have used `default_value_for` in the past, but that might be overkill for this one property.